src/index.html

```diff
@@ -1,13 +1,353 @@
+                <h2 class="bake__subtitle">Donuts</h2>
+                <p class="bake__description">Light multi-layered croissant strewn with cinammon and almond “petals”</p>
+                <a href="#footer" class="bake__button"><h2 class="bake__link--text">Order</h2></a>
+              </div>
+            </div>
+            <div class="
+             grid__item
+             grid__item--tablet-5-8
+             grid__item--desktop-6-8">
+              <div class="bake-container">
+                <img
+                 src="images/section/img44.svg"
+                 class="bake--img"
+                 alt="cakes"
+                >
+                <h2 class="bake__subtitle">Donuts</h2>
+                <p class="bake__description">Oatmeal cookies with golden raisins, orange zest and chocolate chips</p>
+                <a href="#footer" class="bake__button"><h2 class="bake__link--text">Order</h2></a>
+              </div>
+            </div>
+            <div class="
+             grid__item
+             grid__item--tablet-1-4
+             grid__item--desktop-1-3">
+              <div class="bake-container">
+                <img
+                 src="images/section/img5.svg"
+                 class="bake--img"
+                 alt="cakes">
+                <h2 class="bake__subtitle">Cakes</h2>
+                <p class="bake__description">Cake with chocolate salted caramel, Vanilla frutti pebbles inside</p>
+                <a href="#footer" class="bake__button"><h2 class="bake__link--text">Order</h2></a>
+              </div>
+            </div>
+            <div class="
+             grid__item
+             grid__item--tablet-5-8
+             grid__item--desktop-4-6">
+              <div class="bake-container">
+                <img
+                 src="images/section/img6.svg"
+                 class="bake--img"
+                 alt="cakes"
+                >
+                <h2 class="bake__subtitle">Cakes</h2>
+                <p class="bake__description">Oatmeal cookies with golden raisins, orange zest and chocolate chips</p>
+                <a href="#footer" class="bake__button"><h2 class="bake__link--text">Order</h2></a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="page__section about" id="about-us">
+        <div class="about__photo-container">
+          <picture>
+            <source media="(min-width: 1024px)" srcset="./images/section/img7-high.svg">
+            <source media="(min-width: 568px)" srcset="./images/section/img7-tablet.svg">
+            <img
+             src="./images/section/img7.svg"
+             alt="Cookie crumbs"
+             class="about__photo"
+            >
+          </picture>
+        </div>
+        <div class="page-container">
+          <div class="grid grid--desktop">
+            <div class="
+             grid__item
+             grid__item--tablet-1-4
+             grid__item--desktop-1-4">
```

Comment:
You need to check and fix it everywhere
```suggestion
            <div
              class="
               grid__item
               grid__item--tablet-1-4
               grid__item--desktop-1-4"
           >
```